ethereum/contracts/Verifier.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
+
+contract Verifier {
+
+    address public operator;
+
+    constructor(address _operator) public {
+        operator = _operator;
+    }
+
+    function recover(bytes32 _message, bytes memory _signature)
+        public
+        view
+        returns (bool)
+    {
+        address signer = verify(_message, _signature);
+        return operator == signer;
+    }
+
+  /**
+     * @dev Verify checks if the signer's address matches the operator's address
+     */
+    function verify(bytes32 h, bytes memory signature)
```

Comment:
The doc above does not match the return type here, right? 

It would make more sense if we called the logic that returns the address of the signer `recover` and the function that returns a bool if that's the operator `verify`. Is there a reason you swapped those?

Reply:
I agree. The swap was not intentional. Updated in https://github.com/Snowfork/polkadot-ethereum/pull/38/commits/f2fe2d84c111628cbc300984e643275cbcc0c8dd